datalad/dochelpers.py

```diff
@@ -173,8 +173,11 @@ def _parse_parameters(paramdoc):
     entries = __re_spliter1.split(paramdoc)
     result = [(__re_spliter2.split(e)[0].strip(), e)
               for e in entries if e != '']
-    lgr.debug('parseParameters: Given "%s", we split into %s' %
-              (paramdoc, result))
+    lgr.log(
+        1,
```

Comment:
I remember there was some interaction between `1` and "something else" . That is why we hardly ever have logging at that low level

<details>
<summary></summary> 

```shell
$> git grep -A1 '\.log($'
datalad/cmd.py:                lgr.log(
datalad/cmd.py-                    5,
--
datalad/core/local/repo.py:            lgr.log(
datalad/core/local/repo.py-                5,
--
datalad/runner/coreprotocols.py:            lgr.log(
datalad/runner/coreprotocols.py-                5,
--
datalad/runner/protocol.py:        lgr.log(
datalad/runner/protocol.py-            8,
--
datalad/runner/runnerthreads.py:        lgr.log(
datalad/runner/runnerthreads.py-            5,
--
datalad/support/sshconnector.py:            lgr.log(
datalad/support/sshconnector.py-                5,

$> git grep '\.log(1'    
datalad/dataset/gitrepo.py:        lgr.log(1, "Finalizer called on: GitRepo(%s)", path)
datalad/support/annexrepo.py:        lgr.log(1, "Finalizer called on: AnnexRepo(%s)", path)

```
</details>

but IIRC it was relating to custom special remotes and custom implementation of those is gone, so might no longer be pertinent.  I personally would have not made it `1` but something like 5 (HEAVYDEBUG) but I also do not care enough.  Let's proceed.  I will release `maint` with this